demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -286,6 +286,9 @@ def check_dataset_exists(
     process_failed = False
     dataset_set = {data.dataset for data in test_data.data}
     results = []
+    logger.debug(f'Sleeping for 30 seconds to make sure the data set was installed correctly')
+    sleep(30)
```

Comment:
Please pass init_sleep_time as an argument to the function, with default value = 30.

```suggestion
    logger.debug(f'Sleeping for {init_sleep_time} seconds before query for the dataset, to make sure the dataset was installed correctly.')
    sleep(init_sleep_time)
```